iroh-willow/src/form.rs

```diff
@@ -0,0 +1,155 @@
+//! Types for forms for entries
```

Comment:
I like the idea of these, but `form` seems a little confusing as a name, I don't have a better one (yet) though